results/OAEI2025/results.md

```
@@ -0,0 +1,93 @@
+    - TOMATO
+    - LogMap lite
+- Running with exceptions / error, no alignments received
+    - ALIN
+    - MDMapper
+    - OntoMatch (standalone, not possible to run with MELT / SEALS)
+
+## Recall, Precision, F1-Score
+
+| Test Case               |Precision |            |           |          |Recall    |            |           |          |F1-Score  |            |           |          |
+| ----------------------- | -------- | ---------- | --------- | -------- | -------- | ---------- | --------- | -------- | -------- | ---------- | --------- | -------- |
+|                         | Agent-OM   | LogMap   | LogMap Bio | LogMap KG | TIM    | Agent OM   | LogMap   | LogMap Bio | LogMap KG | TIM    | LogMap   | LogMap   | LogMap Bio | LogMap KG | TIM   |
```

markdownlint-cli2 (0.18.1)

Table column count
Expected: 13; Actual: 16; Too many cells, extra data will be missing

(MD056, table-column-count)